### PR TITLE
In 150550599 screening restrictions

### PR DESCRIPTION
**Pull request by AmanShurafa**

### CODE REVIEW

File: spec/factories/screening_factory.rb [hunk 1]
```diff
@@ -65,5 +65,15 @@
...
     end
+
+    trait :sensitive do
+      access_restrictions { 'sensitive' }
+      restrictions_rationale { 'celebrity' }
+    end
+
+    trait :sealed do
+      access_restrictions { 'sealed' }
+      restrictions_rationale { 'family_member_at_risk' }
```
Comment (jgedeon-osi) [inline spec/factories/screening_factory.rb]: Shouldn't this be a block of text from say FFaker::Lorem.paragraphs(2)?

File: spec/requests/api/v1/screenings/submit_spec.rb [hunk 2]
```diff
@@ -56,7 +56,7 @@
     let(:incident_address) { FactoryGirl.build(:address, state: 'UT', type: 'Other') }
     let(:screening) do
       FactoryGirl.create(
-        :screening, :with_started_date, :with_ended_date,
+        :screening, :with_started_date, :with_ended_date, :sensitive,
```
Comment (jgedeon-osi) [inline spec/requests/api/v1/screenings/submit_spec.rb]: Should there be a test for submitting without sensitive?

File: spec/twins/referral_twin_spec.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
 describe ReferralTwin do
   let(:twin) { described_class.new(screening) }
   let(:screening) do
-    FactoryGirl.create :screening,
+    FactoryGirl.create :screening, :sensitive,
```
Comment (jgedeon-osi) [inline spec/twins/referral_twin_spec.rb]: Should there be a test when there is no sensitive flag?

File: spec/factories/screening_factory.rb (hunk 1 above)
Comment (AmanShurafa) [reply spec/factories/screening_factory.rb]: restrictions_rationale is free text but will be drop down field in future. I wanted to use something that is future proof.

File: spec/twins/referral_twin_spec.rb (hunk 3 above)
Comment (AmanShurafa) [reply spec/twins/referral_twin_spec.rb]: Not really, these are 2 other fields with nothing special about them right now. We want to make sure the values are set when passed. Nothing to be done when they're not present

File: spec/requests/api/v1/screenings/submit_spec.rb (hunk 2 above)
Comment (AmanShurafa) [reply spec/requests/api/v1/screenings/submit_spec.rb]: Yeah, there isn't a need for this

File: spec/twins/referral_twin_spec.rb [hunk 4]
```diff
@@ -104,6 +102,24 @@
     expect(twin.participants.count).to eq 0
   end
 
+  it 'sets access restrictions if non is provided' do
```
Comment (jgedeon-osi) [inline spec/twins/referral_twin_spec.rb]: non -> none

**Pull request merged**